app/components/theme-selector.js

```diff
@@ -0,0 +1,14 @@
+import classic from 'ember-classic-decorator';
+import Component from '@ember/component';
+
+@classic
```

Comment:
This should convert to a glimmer component easily....fine to do it later on though.